ynr/apps/wombles/templates/wombles/my_profile.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>My profile</h1>
+    <p>Your username is {{ user.username }} and your contact email is {{ user.email }}.</p>
+    <p>When you create an account we create an API key for you, this lets you use <a href="{% url "api-home" %}">our API</a>.</p>
+    <p>Your API key is <code>{{ user.auth_token.key }}</code>. You can use this by passing <code>?auth_token={{ user.auth_token.key }}</code> to API requests.x</p>
```

Comment:
Is it worth also mentioning the header method/giving a curl example - something like: 
```
curl -X GET candidates.democracyclub.org.uk/api/example/ -H 'Authorization: Token {{ user.auth_token.key }}'
```